receiver/awsxrayreceiver/go.mod

```diff
@@ -5,13 +5,11 @@ go 1.14
 replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/common => ../../internal/common
 
 require (
-	github.com/aws/aws-sdk-go v1.34.5
+	github.com/aws/aws-sdk-go v1.34.8
 	github.com/google/uuid v1.1.1
-	github.com/googleapis/gnostic v0.4.1 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.0.0-00010101000000-000000000000
+	github.com/open-telemetry/opentelemetry-proto v0.4.0
 	github.com/stretchr/testify v1.6.1
-	go.opentelemetry.io/collector v0.8.1-0.20200819173546-64befbcc0564
+	go.opentelemetry.io/collector v0.8.0
```

Comment:
I think it's because I tend to delete packages listed in go.mod and re-run `go test` whenever there's a merge conflict. And on go package registry, the latest collector version is 0.8.0